tools/benchmark/cmd/watch.go

```diff
@@ -169,7 +165,9 @@ func watchFunc(cmd *cobra.Command, args []string) {
 		close(putreqc)
 	}()
 
-	<-recvCompletedNotifier
+	for range streams {
+		<-recvCompletedNotifier
```

Comment:
`atomic.AddInt32(&nrRecvCompleted, 1)` happens inside `for` loop, so I think it should be inside `for` loop?